tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/AssignEntityTests/luis.settings.westus.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "luis": {
+    "app_en_us_lu": "00000000-0000-0000-0000-000000000000"
```

Comment:
>"app_en_us_lu": "00000000-0000-0000-0000-000000000000" [](start = 4, length = 54)

This is minor thing, but generated settings files now look like an object like:
"app_en_us_lu: {
  appId: "0000000",
  version: "0.1"
}
Then the setting reference would add .appId to end.  Would probably be good to update these.